docs/editors-guide/import-terms-for-logical-axioms.md

```diff
@@ -2,79 +2,39 @@
 
 These workflows for adding classes from external ontologies (i.e., GO or CHEBI), which is much more streamlined compared to [MIREOTing](https://github.com/obophenotype/human-phenotype-ontology/wiki/Editor-Guide#mireoting).
 
-### Preferred Instructions
+_Last updated 27-Mar-2024_
 
-_Added 2020-05-06_
-
-1. Open the `src/ontology/imports/*_terms.txt`
-2. Add term to this file
+1. Open the `src/ontology/imports/manual_seed.txt`
+2. Add the IRI of the term(s) you want to add to this document, and save
+    - you can add any IRI, e.g. gene, NCBITaxon,...
 3. In the Terminal, run: `export "MEMORY_GB=15"`
-4. Then run import: `sh run.sh make refresh-merged`
-4. Close Protege and open again
-5. Edit in Protege
-
----
-#### Detailed instructions for adding a new NCBI Taxon identifier 
-
-_Last updated 6-Mar-2024_
-
-1. Fetch the latest changes from `master`
-1. Create a new feature branch for the updated NCBI Taxon additions
-1. Open the `src/ontology/imports/ncbitaxon_terms.txt` file
-1. Add the IRI(s) of interest into the file, e.g. `http://purl.obolibrary.org/obo/NCBITaxon_3052303`
-1. In the Terminal, run: `export "MEMORY_GB=15"`
-1. Run import: `sh run.sh make refresh-merged` (Note: this may take ~2 hours)
-1. Review and commit the updated files, the changed files will be:
-    - `src/ontology/imports/ncbitaxon_terms.txt`
-    - `src/ontology/imports/merged_import.owl`
-1. Publish the branch and create a PR
-1. Once the PR is approved and merged, the new NCBI Taxon can be referenced in logical definitions
-1. Check if the new NCBI Taxon identifier(s) exist in [https://github.com/obophenotype/ncbitaxon/blob/master/subsets/taxon-subset-ids.txt](https://github.com/obophenotype/ncbitaxon/blob/master/subsets/taxon-subset-ids.txt)
+4. Then run import: `sh run.sh make refresh-merged` 
+    - All the imports will be updated, which means that you might see changes in your GitHub diff in the following files
+        - `src/ontology/imports/*_terms.txt`
+        - `src/ontology/imports/merged_import.owl`
+    - The terms added in the "manual_seed.txt" will be added to the appropriate import (e.g human genes will be added to hgnc_terms.txt; NCBITaxon will be added to ncbitaxon_terms.txt), and removed from the "manual_seed.txt" after the import refresh has run. 
+5. Open mondo-edit.obo in protege, and save the ontology (as .obo).
+    - one needs to save the mondo-edit.obo in order for the updates from the import updates (e.g. updated names) to be visible in this file. 
```

Comment:
"this file" = in the mondo-edit.obo

Reply:
I generally try to avoid using demonstrative pronouns (this, that, these, etc.) in documentation to be explicit what "this" is since in general people can have a tendency to refer back to other things earlier in instructions and not just the thing/noun that was on that line.